adders/supabase/config/options.ts

```diff
@@ -0,0 +1,25 @@
+import { colors, defineAdderOptions } from '@svelte-add/core';
+
+export const options = defineAdderOptions({
+	demo: {
+		question: 'Do you want to include demo routes to show protected routes?',
+		type: 'boolean',
+		default: false,
+	},
+	admin: {
+		question: `Do you want to add an Supabase admin client? ${colors.red('Warning: This client bypasses row level security, only use server side.')}`,
```

Comment:
Could it be in a `server` file or directory to enforce it rather than relying on a warning?
```suggestion
		question: `Do you want to add a Supabase admin client? ${colors.red('Warning: This client bypasses row level security, only use server side.')}`,
```

Reply:
The adder creates the admin client in `$lib/server/supabase-admin` so the warning is somewhat redundant. Happy to remove if that is preferred. I included it because there's nothing stopping an unaware user from just moving it out of that directory and inadvertently ruining themself.